feature1/one-more-file.txt

```diff
@@ -0,0 +1,2 @@
+This is the first file that is created inside feature1 folder and second file which is created inside feature1 branch.
```

Comment:
It's really good that you have added this line of code.

Reply:
Thanks for the comment, NJ996Rev!